packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -228,6 +247,12 @@ export interface IFluidContainer<TContainerSchema extends ContainerSchema = Cont
 	 * Dispose of the container instance, permanently disabling it.
 	 */
 	dispose(): void;
+
+	/**
+	 * Allows the host to have the container force to be in read-only mode
```

Comment:
Nit (I always recommend expressing summary documentation in terms of what the associated API member _is_ or _does_. 
```suggestion
	 * Forces the container into read-only mode.
```

Reply:
A `@remarks` block would be a good place to add info about why a user may want to call this.